### PR TITLE
READ_DATABASE / READ_TABLE fix

### PR DESCRIPTION
**Pull request by xBis7**

### CODE REVIEW

**Comment by xBis7** [comment]
@GeorgeJahad

File: hive-agent/src/main/java/org/apache/ranger/authorization/hive/authorizer/RangerHiveAuthorizer.java [hunk 1]
```diff
@@ -3124,7 +3124,12 @@ private RangerHiveAccessRequest buildRequestForAlterTableSetOwnerFromCommandStri
 
 	static void setOwnerUser(RangerHiveResource resource, HivePrivilegeObject hiveObj, IMetaStoreClient metaStoreClient) {
 		if (hiveObj != null) {
-			// resource.setOwnerUser(hiveObj.getOwnerName());
+			LOG.info("gbj setOwner()");
```
Comment (mladjan-gadzic) [inline hive-agent/src/main/java/org/apache/ranger/authorization/hive/authorizer/RangerHiveAuthorizer.java]: Do we need these logs?

Comment (xBis7) [reply hive-agent/src/main/java/org/apache/ranger/authorization/hive/authorizer/RangerHiveAuthorizer.java]: No, but they make debugging easier.

Comment (xBis7) [reply hive-agent/src/main/java/org/apache/ranger/authorization/hive/authorizer/RangerHiveAuthorizer.java]: Once, I finish I can clean them up.

Comment (mladjan-gadzic) [reply hive-agent/src/main/java/org/apache/ranger/authorization/hive/authorizer/RangerHiveAuthorizer.java]: Let's do that. I'll approve it so you can go ahead and merge when you're done.

**Review by mladjan-gadzic — APPROVED** [review]
@xBis7 thanks for the work!

**Pull request merged**